src/app/services/indicator.service.ts

```diff
@@ -10,15 +10,16 @@ import { apiHost } from '../constants';
 
 /*
  * Indicator Service
- * Returns climate indicators. Used by the sidebar.
+ * Returns climate indicators. Used by the sidebar and to retrieve detail data for charts.
  */
 @Injectable()
 export class IndicatorService {
     constructor(private apiHttp: ApiHttp) {}
 
     public getData(options: IndicatorQueryOpts) {
-        const url = `${apiHost}/api/climate-data/${options.city.id}/${options.scenario.name}` +
-                  `/indicator/${options.indicator.name}/`;
+
+        const getDetailUrl = `${apiHost}/api/climate-data/${options.city.id}/${options.scenario.name}` +
```

Comment:
Reason why has been rebased away. Was going to maybe put the URL build in a separate, non-async method but decided it wasn't worth it. Sure, can rename back.